src/plugins/graph/components/legend/legend-registration.ts

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from "react";
+
+import { colorIdsOfLayerLegend, heightOfLayerLegend, LayerLegend, layerLegendType } from "./layer-legend";
+import { ColorIdListFunction, ILegendPartProps, LegendHeightFunction } from "./legend-types";
+
+interface IMultiLegendPart {
+  component: FunctionComponent<ILegendPartProps>;
+  getHeight: LegendHeightFunction;
+  getColorIdList: ColorIdListFunction;
```

Comment:
`getColorIdList` suggests to me that it should return the IDs of colors, which it doesn't.  Maybe eventually these elements will get shapes or line styles to differentiate them as well as colors, too.  So maybe this should be named something like `getLegendItemIDs` or something like that?  Not a great name I know.